phoronix/run_phoronix.sh

```diff
@@ -119,31 +119,13 @@ if [ $show_usage -eq 1 ]; then
 fi
 
 test_tools/package_tool --update
-test_tools/package_tool --packages php73-cli.x86_64,php73-common.x86_64,php73-xml.x86_64
+php_cli=`dnf list | grep  php  |  grep cli | awk '{print $1}'`
+php_common=`dnf list | grep php  | grep common | awk '{print $1}'`
+php_xml=`dnf list | grep php  | grep xml | awk '{print $1}'`
```

Comment:
What happens if any of these queries result in an empty string?

I think we should error out if we aren't able to find or get the needed packages